README.md

```diff
@@ -1,78 +1,248 @@
+print(ucare_file.cdn_url)  # file URL, e.g.: https://ucarecdn.com/640fe4b7-7352-42ca-8d87-0e4387957157/
+```
 
-    class Candidate(models.Model):
-        photo = ImageField(blank=True, manual_crop="")
+And what about information about the file?
 
+```python
+from pprint import pprint
+
+pprint(ucare_file.info)
+
+# {'appdata': None,
+#  'content_info': {'image': {'color_mode': <ColorMode.RGB: 'RGB'>,
+#                             'datetime_original': datetime.datetime(2023, 3, 10, 16, 23, 15),
+#                             'dpi': (72, 72),
+#                             'format': 'JPEG',
+#                             'geo_location': None,
+#                             'height': 4516,
+#                             'orientation': 1,
+#                             'sequence': False,
+#                             'width': 3011},
+#                   'mime': {'mime': 'image/jpeg',
+#                            'subtype': 'jpeg',
+#                            'type': 'image'},
+#                   'video': None},
+#  'datetime_removed': None,
+#  'datetime_stored': datetime.datetime(2024, 2, 16, 14, 44, 29, 637342, tzinfo=TzInfo(UTC)),
+#  'datetime_uploaded': datetime.datetime(2024, 2, 16, 14, 44, 29, 395043, tzinfo=TzInfo(UTC)),
+#  'is_image': True,
+#  'is_ready': True,
+#  'metadata': {},
+#  'mime_type': 'image/jpeg',
+#  'original_file_url': 'https://ucarecdn.com/640fe4b7-7352-42ca-8d87-0e4387957157/samplefile.jpeg',
+#  'original_filename': 'sample-file.jpeg',
+#  'size': 3518420,
+#  'source': None,
+#  'url': 'https://api.uploadcare.com/files/640fe4b7-7352-42ca-8d87-0e4387957157/',
+#  'uuid': UUID('640fe4b7-7352-42ca-8d87-0e4387957157'),
+#  'variations': None}
+```
 
-    # optional. provide advanced widget options: https://uploadcare.com/docs/uploads/widget/config/#options
-    class CandidateForm(forms.Form):
-        photo = ImageFormField(widget=FileWidget(attrs={
-            'data-cdn-base': 'https://cdn.super-candidates.com',
-            'data-image-shrink': '1024x1024',
-        }))
+A whole slew of different file operations are available. Do you want to crop your image, but don't want important information (faces, objects) to be cropped? You can do that with content-aware (“smart”) crop:
 
+```python
+from pyuploadcare.transformations.image import ImageTransformation, ScaleCropMode
+
+# These two function calls are equivalent
+ucare_file.set_effects("scale_crop/512x512/smart/")
+ucare_file.set_effects(ImageTransformation().scale_crop(512, 512, mode=ScaleCropMode.smart))
+
+print(ucare_file.cdn_url)  # https://ucarecdn.com/640fe4b7-7352-42ca-8d87-0e4387957157/-/scale_crop/512x512/smart/
 ```
 
-![](https://ucarecdn.com/f0894ef2-352e-406a-8279-737dd6e1f10c/-/resize/800/manual_crop.png)
+There’s a lot more to uncover. For more information please refer to the [documentation](#documentation).
 
-## Documentation
+### Django integration
+
+<!-- TODO: The idea of this example is to show how upload and deliver an image -->
```

Comment:
What should we do with this *TODO*?

Reply:
Anything. Example is very much up to you. You know our products well enough. :-)